i18n/es.jsonp.js

```diff
@@ -19,8 +19,14 @@ document.localeJson = {
       "configure-title": "Configuración",
       "configure-about-usage-link": "Información (Uso)",
       "configure-btn-library": "Biblioteca ZIM",
+      "configure-btn-folderselect": "Seleccione carpeta",
+      "configure-btn-rescan": "Volver a escanear",
+      "configure-about-rescan-tip": "Vuelve a escanear las tarjetas SD y la memoria interna",
+      "configure-select-file-numbers": "archivo(s) encontrados en la ubicación seleccionada. ",
```

Comment:
You also need to add parentheses round the plural form of the adjective, i.e. "encontrado(s)"